.github/workflows/sync-output-repos.yml

```diff
@@ -57,7 +57,7 @@ jobs:
 
   push-editors:
     name: "Push to editor output repos (${{ matrix.variant }})"
-    if: github.ref_type == 'branch'
+    if: github.ref_type == 'tag'
```

Comment:
Could you instead remove the guard on both jobs and change the `on` part to only run on tags?

Reply:
can do!